provision/acc_provision/acc_provision.py

```diff
@@ -172,6 +172,7 @@ def config_default():
             "interface_mtu": None,
             "second_kubeapi_portgroup": False,
             "disable_wait_for_network": False,
+            "wait_for_network_duration": 210,
```

Comment:
Per earlier suggestion, can we change this to "duration_wait_for_network" (although I know the current name reads better)? I would like to keep the consistency with the naming convention of the related configuration "disable_wait_for_network". Else, we could change "disable_wait_for_network" to "wait_for_network_disable" (but then it will have to be changed in a number of places).